### ISSUE TITLE
XML export doesn't work in 5.0.0

### ISSUE THREAD

**Issue by maximumspatium** [Audiveris/audiveris#24]
Steps to reproduce:

- open "allegretto.png" from `data/examples` in Audiveris 5.0.0
- transcribe it
- save the result as MusicXML using "Export Sheet as..."
- open the saved MusicXML in Musescore

Although the resulting MusicXML is valid, it doesn't contain any music.

The commit causing the issue has been identified using `git bisect`:

first bad commit: [0b23b0bfa0f765c0a0966eb4137c44868d798936](https://github.com/Audiveris/audiveris/commit/0b23b0bfa0f765c0a0966eb4137c44868d798936) Handling of Transcribable for book & sheet

**Comment by hbitteur** [comment]
Strange, I just tried to reproduce the problem (using master branch by default), and everything seems fine.
I get an output allegretto.mxl file of 4919 bytes. Fed into MuseScore and into Finale, music looks and sounds indeed OK. There are a few words/characters uncorrectly OCR'ed but with no impact on music.

The de-compressed allegretto.xml file contains 2366 lines with a correct score-partwise structure. Could you elaborate on "_Although the resulting MusicXML is valid, it doesn't contain any music._"

**Comment by maximumspatium** [comment]
> Could you elaborate on "Although the resulting MusicXML is valid, it doesn't contain any music."

Sure. Below a short excerpt of the Audiveris log window:

```
End of sheet step PAGE
Score Allegretto exported to /Users/*****/Documents/Allegretto.xml
```

The resulting MusicXML is [here](https://pastebin.com/Bs9zLYqN).

**Comment by hbitteur** [comment]
I noticed these lines:

>     <miscellaneous>
>       <miscellaneous-field name="source-file">/Users/maxim/Documents/Development/audiveris-ng/data/examples/allegretto.png</miscellaneous-field>
>     </miscellaneous>

Before the closing /miscellaneous, you should have the following line:

>     <miscellaneous-field name="source-sheet-1">1 2 3</miscellaneous-field>

which tells that sheet number 1 contains systems 1,2 and 3.
Since this line is not present in the XML output, I suspect that the sheet entity is NOT present in the containing score entity.
Could you save the book and check in the **allegretto.omr** file (which is just a zip file) what the content of the **book.xml** file is? (it is a very small file, about a dozen of lines, you can post its content on line here).
Check the lines dealing with sheet entities and score entities.

**Comment by maximumspatium** [comment]
> Could you save the book and check in the allegretto.omr file (which is just a zip file) what the content of the book.xml file is?

Here we go:

```
<?xml version="1.0" ?>
<book software-version="5.0.0" software-build="bd6f204bd" path="/Users/maxim/Documents/Development/audiveris-ng/data/examples/allegretto.png">
  <sheet number="1">
    <steps>LOAD BINARY SCALE GRID HEADERS STEM_SEEDS BEAMS LEDGERS HEADS STEMS REDUCTION CUE_BEAMS TEXTS MEASURES CHORDS CURVES SYMBOLS LINKS RHYTHMS PAGE</steps>
    <page id="1" movement-start="true" delta-measure-id="17"></page>
  </sheet>
  <score id="1">
    <page sheet-number="1" sheet-page-id="1"></page>
  </score>
</book>
```

**Comment by maximumspatium** [comment]
I was able to further narrow down the issue:

- "Export Book as..." from the Book menu produces **working** MusicXMLs.
- "Export Sheet as..." from the Sheet menu produces **crippled** MusicXMLs.

**Comment by hbitteur** [comment]
Fixed.

**Comment by maximumspatium** [comment]
It would be good to know the precise commit that fixes the issue...

**Comment by hbitteur** [comment]
Commit for fix was 3cbbc559e9af5ccc29b1e14f1f03f559c1b88a71
